app/src/services/datasetService.js

```diff
@@ -63,33 +64,48 @@ class DatasetService {
+                                alertType: datasetQuery.type,
+                                alertBeginDate: datasetQuery.lastSentDate.toISOString().slice(0, 10),
+                                alertEndDate: new Date().toISOString().slice(0, 10),
+                                alertResult: result.data[0].value
+                            };
+
+                            // Execute EMAIL notification - sending an email
                             if (subscription.resource.type === 'EMAIL') {
-                                const areaName = subscription.params.area ? await DatasetService.getAreaName(subscription.params.area) : '';
-                                const datasetName = metadata[0].attributes.info.name || dataset.attributes.name || dataset.name;
-                                const data = {
-                                    subject: subscription.name ? subscription.name : `${datasetQuery.type} in ${areaName} above ${datasetQuery.threshold}`,
-                                    datasetName,
-                                    datasetId: datasetQuery.id,
-                                    datasetSummary: metadata[0].attributes.info.functions,
-                                    areaId: subscription.params.area ? subscription.params.area : '',
-                                    areaName,
-                                    alertName: subscription.name ? subscription.name : `${datasetQuery.type} in ${areaName} above ${datasetQuery.threshold}`,
-                                    alertType: datasetQuery.type,
-                                    alertBeginDate: datasetQuery.lastSentDate.toISOString().slice(0, 10),
-                                    alertEndDate: new Date().toISOString().slice(0, 10),
-                                    alertResult: result.data[0].value
-                                };
                                 logger.debug('Sending mail with data', data);
                                 let template = 'dataset-rw';
                                 if (subscription.env && subscription.env !== 'production') {
                                     template += `-${subscription.env}`;
                                 }
                                 MailService.sendMail(template, data, [{ address: subscription.resource.content }], 'rw'); // sender='rw'
+                            }
 
-                            } else {
-                                // @TODO resource.type === 'WEBHOOK'?
+                            // Execute URL notification - POSTing to webhook
+                            if (subscription.resource.type === 'URL') {
+                                // POST to URL configured in subscription.resource.content
+                                request.post(subscription.resource.content, { json: data }, (error, res, body) => {
+                                    if (res && res.statusCode === 200) {
```

Comment:
`statusCode === 200` => `statusCode < 400`

Reply:
Treating `statusCode < 400` as a successful response, and the returned statusCode is logged :+1: